cmd/podman/common/create.go

```diff
@@ -389,7 +389,7 @@ func GetCreateFlags(cf *ContainerCLIOpts) *pflag.FlagSet {
 	createFlags.StringVar(
 		&cf.Restart,
 		"restart", "",
-		`Restart policy to apply when a container exits ("always"|"no"|"on-failure")`,
+		`Restart policy to apply when a container exits ("always"|"no"|"on-failure"|"unless-stopped"|"on-success"|"on-abnormal"|"on-watchdog"|"on-abort")`,
```

Comment:
alphabetize please:
```suggestion
		`Restart policy to apply when a container exits ("always"|"no"|"on-abnormal"|"on-abort"|"on-failure"|"on-success"|"on-watchdog"|"unless-stopped")`,
```

Reply:
`podman create/run --restart` only supports `always`, `no`, `on-failure` and `unless-stopped`. At least according to our documentation (`man podman-create`)